src/runtime/runtime.js

```diff
@@ -515,6 +515,13 @@ export class Runtime {
       )
     );
   }
+
+  /** @override */
+  showBestAudienceAction() {
+    console.error('Forbidden call, under implementation.');
```

Comment:
Maybe use `debugLog` for these for now. Linter doesn't seem to like the `console` messaging

Reply:
`warn` (from "src/utils/log.js") might be even better